config/games/simonionburg.ts

```diff
@@ -0,0 +1,34 @@
+import { ServerGame } from "../../types/types";
+import { randomBytes } from "crypto";
+
+const game: ServerGame = {
+	id: "simonionburg",
+	name: "Simonionburg",
+	author: "Simon Weisse",
+	basedOn: {
+		game: "Quacks of Quedlinburg and Dominion (losely)",
+		author: "Wolfgang Warsch (Quacks), Donald X. Vaccarino (Dominion)",
+		link: "https://boardgamegeek.com/boardgame/244521/quacks-quedlinburg", // https://dominion.games/ for a second game link /shrug
```

Comment:
I don't think it's common to base a game off of two games, but not being able to link both of them is a conundrum.  Not a big deal, but thought I'd mention it.